app/0.14.x/configuration.md

```diff
@@ -1035,7 +1035,6 @@ record types. The `LAST` type means the
 type of the last successful lookup (for the
 specified name). The format is a (case
 insensitive) comma separated list.
-Remove any record types that Kong should ignore. 
```

Comment:
@Tieske can you clarify why this comment was removed? 

If you want Kong to ignore certain record types, can you remove them from this list? If not, then is it a requirement that all 4 record types remain present in this list?

Reply:
<opinion> too detailed </opinion>